src/ids/quickId.ts

```diff
@@ -243,3 +269,39 @@ export async function quickId(
 
   return (await Promise.all(idPromises)).flat();
 }
+
+async function getRecentDocumentByPosition(
+  db: PouchDB.Database<EitherPayload>,
+  position: number,
```

Comment:
I'm not a fan of position as the variable name here. I would recommend renaming this to `getRecentDocument` and then use indexFromEnd or something to be more explicit